app/components/collectionShow/relatedPaperInCollectionShow.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { withStyles } from "../../helpers/withStylesHelper";
+import CollectionAPI from "../../api/collection";
+import { Paper } from "../../model/paper";
+import PaperItem from "../common/paperItem";
+import ArticleSpinner from "../common/spinner/articleSpinner";
+import ActionTicketManager from "../../helpers/actionTicketManager";
+const styles = require("./relatedPaperInCollectionShow.scss");
+
+interface RelatedPaperInCollectionShowProps {
+  collectionId: number;
+}
+
+const RelatedPaperInCollectionShow: React.FunctionComponent<RelatedPaperInCollectionShowProps> = props => {
+  const { collectionId } = props;
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [relatedPapers, setRelatedPapers] = React.useState<Paper[]>([]);
+
+  React.useEffect(
+    () => {
+      setIsLoading(true);
+      CollectionAPI.getRelatedPaperInCollection(collectionId).then(result => {
+        if (result.content && result.content.length !== 0) {
+          ActionTicketManager.trackTicket({
```

Comment:
it's not a good pattern.
because it always tracks the view count even if the paper isn't exposed to a user.

Reply:
Trying to use `IntersectionObserver`